packages/api/cms-api/src/common/enum/comet-permission.enum.ts

```diff
@@ -0,0 +1,18 @@
+import { registerEnumType } from "@nestjs/graphql";
+
+export enum CometPermission {
```

Comment:
As discussed, please rename to CorePermission or LibraryPermission. Also, should we move it to src/user-permissions?

Reply:
done